FreeRTOS/FreeRTOS-Plus-TCP
# Add Unit Tests for FreeRTOS_DNS_Cache

Add Unit Tests for FreeRTOS_DNS_Cache

Description
-----------
This change adds the Unit test case for this file along with 100% coverage.

Test Steps
-----------
cmake -S test/unit-test -B test/unit-test/build/ -DCMAKE_BUILD_TYPE=Debug
make -C test/unit-test/build/ all

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
